symplyphysics/laws/kinematic/damped_oscillations/critical_damping.py

```diff
@@ -0,0 +1,91 @@
+from sympy import Eq, exp, dsolve, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.definitions import damped_harmonic_oscillator_equation as damped_eqn
+
+# Description
+## In the presence of a damping force in the oscillating system, the system's behaviour
+## depends on the value of the damping ratio. When it is equal to 1, the system returns
+## to equilibrium as quickly as possible without oscillating, but overshoot can occur if
+## initial velocity is nonzero. This behaviour is also called critical damping.
+
+# Law: x(t) = exp(-omega*t) * (x0 + (v0 + x0*omega)*t)
+## x(t) - position of critically damped oscillator
+## t - time
+## omega - oscillator frequency in an undamped state
+## x0 - oscillator's initial position
+## v0 - oscillator's initial velocity
+
+displacement = Function("displacement", units.length)
+time = Symbol("time", units.time, positive=True)
+undamped_angular_frequency = Symbol("undamped_angular_frequency", angle_type / units.time, positive=True)
+initial_position = Symbol("initial_position", units.length)
+initial_velocity = Symbol("initial_velocity", units.velocity)
+
+c1 = initial_position
+c2 = initial_velocity + c1 * undamped_angular_frequency
+
+law = Eq(
+    displacement(time),
+    exp(-1 * undamped_angular_frequency * time) * (c1 + c2 * time),
```

Comment:
Write formula like in description. Do not use `c1` and `c2` here.

Reply:
Okay, it's just that the formula will look too big